Content.Server/AlertLevel/AlertLevelOnPressSystem.cs

```diff
@@ -36,27 +37,12 @@ public override void Initialize()
     {
         base.Initialize();
 
-        SubscribeLocalEvent<ChangeAlertLevelComponent, ActivateInWorldEvent>(OnActivated);
+        SubscribeLocalEvent<AlertLevelOnPressComponent, ActivateInWorldEvent>(OnButtonPressed);
     }
 
-    public void OnActivated(Entity<ChangeAlertLevelComponent> ent, ref ActivateInWorldEvent args)
+    private void OnButtonPressed(Entity<AlertLevelOnPressComponent> ent, ref ActivateInWorldEvent args)
```

Comment:
still need to actually make ButtonPressedEvent and change it

also rename the component and move to server

Reply:
wait what. i thought i was supposed to delete the server version and move it entirely to shared